lib/model.js

```diff
@@ -987,18 +988,18 @@ Model.prototype.$__where = function _where(where) {
 };
 
 /**
- * Removes this document from the db. Equivalent to `.remove()`.
+ * Delete this document from the db.
  *
  * #### Example:
  *
- *     product = await product.deleteOne();
+ *     await product.deleteOne();
  *     await Product.findById(product._id); // null
```

Comment:
why was this changed but the next line not modified?

Reply:
Why would the `findById()` line be modified? The assumption is that here's a `product` document floating around that has an `_id`, so there's no need to do `product = await product.deleteOne()`. Just `await product.deleteOne()` should be sufficient.